tests/bootstrap.php

```diff
@@ -5,4 +5,26 @@
  */
 
 $loader = require __DIR__ . "/../vendor/autoload.php";
-$loader->addPsr4( 'YOURLS\\', __DIR__ );
+
+
+/**
+* Returns the name of a temporary file
+*/
+function temp_filename() {
+    $tmp_dir = realpath(__DIR__);
+    return tempnam($tmp_dir, 'pomo');
+}
+
+function replace_r_n($text) {
+    return str_replace("\r\n", "\n", $text);
+}
+
+// PHPUnit 6 compatibility for previous versions
+if (class_exists('PHPUnit\Runner\Version' ) && version_compare( PHPUnit\Runner\Version::id(), '6.0', '>=' ) ) {
```

Comment:
Hmm... indeed, this works when you install PHPUnit via Composer, which downloads the latest 4.8, instead of using the built-in PHPUnit for each PHP build, which is --obviously-- [outdated and not compatible](https://travis-ci.org/ozh/phpass/jobs/233439542#L187). Makes sense :)